students/jeewon-shin/westagram/user/views.py

```diff
@@ -0,0 +1,25 @@
+import json
+from django.http  import JsonResponse
+
+from django.views import View
+from user.models  import User
+
+class SignupView(View):
+    def post(self, request):
+        try:
+            data  = json.loads(request.body)
+            if '@' and '.' not in data['email']:
+                return JsonResponse({'MESSGAGE': 'This is not valid email.'}, status=400)
```

Comment:
- 리턴 메시지는 클라이언트가 보는 것이 아닌고 프론트엔드가 보는 메시지 입니다. 굳이 문장으로 안 하시고, 간단하게 표현해도 좋을 것 같습니다.
- 메시지 형식은 통일하는 것이 좋습니다. 다 대문자로 썼으면 대문자로만, _을 사용해서 단어을 이였다면 계속 _을 사용하는 것처럼요
```suggestion
                return JsonResponse({'MESSGAGE': 'Invalid email.'}, status=400)
```